scripts/parser-tests/typescript/allowlist.txt

```diff
@@ -221,6 +222,7 @@ exportSpecifierAndLocalMemberDeclaration.ts
 exportStarFromEmptyModule.ts
 exportStarNotElided.ts
 expressionsForbiddenInParameterInitializers.ts
+extendedUnicodePlaneIdentifiers.ts
```

Comment:
I have no idea why this test is in allowlist:

[Source](https://github.com/microsoft/TypeScript/blob/master/tests/cases/compiler/extendedUnicodePlaneIdentifiers.ts)

Babel parser parses successfully, as expected. ([repl](https://babel.dev/repl#?browsers=&build=&builtIns=false&corejs=false&spec=false&loose=true&code_lz=MYewdgzgLgBIrBuC0dmBeGAWA3AKFJWdABHZRgFZtcIQAbAUwDoqQBzACkJgGp4EBKDGAPQCYATiw4qAQwgQYAaRgBvLDFUwAxImKYVazUVRksAX3FCYjAO40ATjAAcMAEYBLKLMYgA1hZdeaMADkkoE44NAwgIYIxABs5OHU9Iys0QBUqVF8ZsJWtg7ObrKSNjRggbCSMABuklQuACYwAK5gLqD1AcAAFsWSwFC2YXgwgEgIxADs8ZCJDMwsY-mjWVjmlQC2LgAeADQwAGYgdkwgII1rNNJNJUMRI5HsqJM3M8nz90SLd4TLFLDvY48ppRaLMUoQFhl_j9wvgCHcJkCXnNwWkMijocM4IBHnbggDuduCAV53CXBAB87xAARHBAAC7pLggG-d4mAW53ydgbrBAPY0gEcaCneFmCYQAIQAsgAFcS_GCAGA3ABw7ksAnDuSwBcOzyvHzzABlUXimEwQCiG4Bavd1gDq93WAer2VWrhOqAJJitl6_VKzl4i2siVOvEG13a4bO3EGpXe-2ex2K7mocm8jBAA&debug=false&forceAllTransforms=false&shippedProposals=true&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=false&presets=typescript&prettier=false&targets=&version=7.14.4&externalPlugins=))